src/autoscaler/eventgenerator/aggregator/metricPoller.go

```diff
@@ -104,17 +102,17 @@ func (m *MetricPoller) doAggregate(appId string, metricType string, metrics []*m
 		avgAppMetric = &model.AppMetric{
 			AppId:      appId,
 			MetricType: metricType,
-			Value:      0,
+			Value:      -1,
 			Unit:       "",
-			Timestamp:  0,
+			Timestamp:  time.Now().UnixNano(),
```

Comment:
We aggregate a set of metrics and should get an AVG metric. The timestamps of this metric set range from T1 to Tn, and for the AVG one , what its timestamp should be? T1 or Tn or NOW(). Also, if the set is empty and we will still save a nil-value metric, and there is no T1 and Tn. So here I set the AVG one's timestamp to NOW().